website/docs/r/cis_dns_record.html.markdown

```diff
@@ -8,61 +8,350 @@ description: |-
-* `ttl`-(Optional,int) TTL of the record. It should be automatic(i.e ttl=1) if the record is proxied. Terraform provider takes ttl in unit seconds.Therefore, it starts with value 120.
+- `cis_id` - (Required,string) The ID of the CIS service instance
+- `domain_id` - (Required,string) The ID of the domain to add the DNS record to.
+- `type` - (Required, string) The type of the DNS record to be created. Supported Record types are: A, AAAA, CNAME, LOC, TXT, MX, SRV, SPF, NS, CAA.
+- `name` - (Required, string) The name of a DNS record.
+- `content` - (Optional,string) The (string) value of the record, e.g. "192.168.127.127". Either this or `data` must be specified
+- `ttl`-(Optional,int) TTL of the record. It should be automatic(i.e ttl=1) if the record is proxied. Terraform provider takes ttl in unit seconds. Therefore, it starts with value 120.
+- `priority` - (Optional, int) The priority of the record. Mandatory field for SRV record type.
+- `data` - (Optional,map) Map of attributes that constitute the record value. Only for LOC, CAA and SRV record types. Either this or `content` must be specified
+  - `weight` - (Optional, int) The weight of distributing queries among multiple target servers. Mandatory field for SRV record
+  - `port` - (Optional, int) The port number of the target server. Mandatory field for SRV record.
+  - `service` - (Optional, int) The symbolic name of the desired service, start with an underscore (\_). Mandatory field for SRV record.
+  - `protocol` - (Optional, int) The symbolic name of the desired protocol. Madatory field for SRV record.
+  - `altitude` - (Optional, int) The LOC altitude. Mondatory field for LOC record.
+  - `size` - (Optional, int) The LOC altitude size. Mondatory field for LOC record.
+  - `lat_degrees` - (Optional, int) The LOC latitude degrees. Mondatory field for LOC record.
+  - `lat_direction` - (Optional, string) The LOC latitude direction ("N", "E", "S", "W"). Mondatory field for LOC record.
+  - `lat_minutes` - (Optional, int) The LOC latitude minutes. Mondatory field for LOC record.
+  - `lat_seconds` - (Optional, int) The LOC latitude seconds. Mondatory field for LOC record.
+  - `long_degrees` - (Optional, int) The LOC Longitude degrees. Mondatory field for LOC record.
+  - `long_direction` - (Optional, string) The LOC longitude direction ("N", "E", "S", "W"). Mondatory field for LOC record.
+  - `long_minutes` - (Optional, int) The LOC longitude minutes. Mondatory field for LOC record.
+  - `long_seconds` - (Optional, int) The LOC longitude seconds. Mondatory field for LOC record.
+  - `precision_horz` - (Optional, int) The LOC horizontal precision. Mondatory field for LOC record.
+  - `precision_vert` - (Optional, int) The LOC vertical precision. Mondatory field for LOC record.
+- `priority` - (Optional,int) The priority of the record
+- `proxied` - (Optional,bool) Whether the record gets CIS's origin protection; defaults to `false`.
 
 ## Attributes Reference
```

Comment:
Please refer only computed fields in attribute reference..

Reply:
done